doc/doxygen/src/coc-faq.md

```diff
@@ -0,0 +1,72 @@
+# FAQ  {#coc-faq}
+
+This FAQ attempts to address common questions and concerns around the RIOT
+community's future [Code of Conduct][CoC]. If you still have questions after
```

Comment:
```suggestion
community's [Code of Conduct][CoC]. If you still have questions after
```

Or we leave it as is to be a perfect copy of the Wiki page and change it in a follow-up PR.

Reply:
In the past, I updated it just with a follow-up commit. So I would just do it here.